destinations/logos/last9.svg

```diff
@@ -0,0 +1 @@
9.2 9.2 0 0 0-3.136.541c-1.001.36-1.863.858-2.581 1.49l-1.046-1.735c.87-.742 1.917-1.315 3.136-1.72a12.199 12.199 0 0 1 3.855-.606c2.248-.001 3.968.562 5.166 1.685zm-2.613 13.102c.937-.6 1.634-1.468 2.09-2.603v-2.817h-5.39c-2.94 0-4.41 1.026-4.41 3.079 0 1.004.381 1.796 1.143 2.375.761.578 1.829.868 3.2.868 1.309 0 2.43-.3 3.367-.902zM40.528 32.357c-1.274-.403-2.27-.91-2.989-1.522L38.585 29c.719.568 1.623 1.032 2.712 1.392 1.089.36 2.222.541 3.398.541 1.569 0 2.728-.246 3.479-.737.752-.49 1.128-1.184 1.128-2.08 0-.633-.207-1.13-.622-1.49-.413-.36-.936-.633-1.569-.82-.632-.185-1.47-.365-2.516-.54-1.394-.262-2.516-.53-3.365-.803a4.935 4.935 0 0 1-2.172-1.392c-.599-.655-.899-1.562-.899-2.719 0-1.44.6-2.62 1.797-3.537 1.197-.918 2.864-1.375 4.998-1.375 1.112 0 2.223.147 3.334.442 1.11.295 2.025.683 2.743 1.162l-1.012 1.868c-1.417-.983-3.105-1.474-5.065-1.474-1.48 0-2.597.261-3.348.786-.752.524-1.128 1.212-1.128 2.063 0 .655.213 1.18.637 1.573.425.393.953.683 1.584.868.631.186 1.502.377 2.613.573 1.372.261 2.472.524 3.3.785s1.536.71 2.124 1.344.882 1.507.882 2.62c0 1.505-.627 2.701-1.878 3.586-1.253.884-2.99 1.326-5.211 1.326a13.09 13.09 0 0 1-4-.605zM65.864 31.751c-.435.394-.975.694-1.617.901s-1.312.312-2.01.312c-1.611 0-2.853-.437-3.724-1.31s-1.307-2.107-1.307-3.702V17.537h-3.07v-1.965h3.07v-3.767h2.32v3.767h5.227v1.965h-5.227V27.82c0 1.027.255 1.808.768 2.342.512.535 1.247.802 2.205.802a4.26 4.26 0 0 0 1.389-.229 3.446 3.446 0 0 0 1.16-.655z"/><path fill-rule="evenodd" d="m97.905 21.961-1.99-.194a18.255 18.255 0 0 0 0-3.534l1.99-.194a20.24 20.24 0 0 1 0 3.922zm-.761-7.769-1.914.58a17.887 17.887 0 0 0-1.351-3.257l1.763-.945a19.875 19.875 0 0 1 1.502 3.622zm-3.683-6.88-1.545 1.27a18.121 18.121 0 0 0-2.498-2.497l1.27-1.546a20.123 20.123 0 0 1 2.773 2.773zM87.43 2.358l-.945 1.763a17.88 17.88 0 0 0-3.257-1.35l.58-1.915c1.265.383 2.476.889 3.622 1.502zM79.96.095l-.194 1.99a18.246 18.246 0 0 0-3.534 0l-.194-1.99a20.242 20.242 0 0 1 3.922 0zm-7.769.761.58 1.914a17.88 17.88 0 0 0-3.257 1.351l-.945-1.763A19.877 19.877 0 0 1 72.192.856zm-6.88 3.683 1.27 1.546a18.121 18.121 0 0 0-2.497 2.497l-1.546-1.27a20.123 20.123 0 0 1 2.773-2.773zm3.258 33.103.945-1.763c1.03.552 2.12 1.006 3.257 1.35l-.58 1.915a19.875 19.875 0 0 1-3.622-1.502zm7.469 2.263.194-1.99a18.255 18.255 0 0 0 3.534 0l.194 1.99a20.24 20.24 0 0 1-3.922 0zm7.769-.761-.58-1.914a17.877 17.877 0 0 0 3.257-1.351l.945 1.763a19.875 19.875 0 0 1-3.622 1.502zm6.88-3.683-1.27-1.545a18.12 18.12 0 0 0 2.497-2.498l1.546 1.27a20.126 20.126 0 0 1-2.773 2.773zm4.954-6.031-1.763-.945a17.885 17.885 0 0 0 1.35-3.257l1.915.58a19.875 19.875 0 0 1-1.502 3.622z" clip-rule="evenodd"/><path d="M70 16.116c0 4.555 3.13 7.71 7.54 7.71.664 0 1.292-.115 1.874-.265L73.642 32h5.062l5.516-8.116c2.048-3.029 3.037-5.375 3.06-7.872C87.292 10.925 83.289 8 78.553 8 73.584 8 70 11.249 70 16.116zm4.84-.174c0-2.347 1.537-4.035 3.736-4.035 2.21 0 3.77 1.688 3.77 4.058 0 2.347-1.56 4.012-3.77 4.012-2.2 0-3.735-1.665-3.735-4.035z"/></g></svg>
```

Comment:
I think the image will show up as a square (same hight and length).
Do you have the logo also as icon format?

<img width="37" alt="image" src="https://github.com/user-attachments/assets/39188318-421b-4bd2-b1ee-c17e291066f9">